index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Gourment Garden kart</title>
+</head>
+<link rel="stylesheet" href="/site.css">
+<!-- font awesome icon -->
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<body>
+    <nav>
+        <div class="nav-responsive">
+            <img src="/logo/logo-Bengaluru_small.png" alt="image" class="logo" id="logo">
+            <div class="access">
+                <a href="#" id="cart"><i class="fa fa-shopping-cart fa-2x"></i></a>
+                <a href="#" id="map"><i class="fa fa-map-marker fa-2x"></i></a>
+            </div>
+        </div>      
+    </nav>
+    <div class="Header">
+        <h2>Freshly Cut</h2>
+         </div>
+         <ul class="content">
+          <li><a class="first" href="#home">ALL</a></li>
+         <li><a href="#news">VEGETABLES</a></li> 
+         <li><a href="#contact">LEAFY GREENS</a></li>
+         <li><a hreg="#contact1">FRESHLY CUT</a></li>
+         <li><a class="last" href="#about">DRY FRUITS</a></li>
+       </ul>
+          <div class="row">
+        <div class="column">
+          <div class="card">
+              <div class="top-left">Organic</div>
+            <center><img src="/images1/Broclii.jpg"height="80px" width="150px;"><center>
+            <p class="title_item">Broclii-Florets</p>
+            <p><span class="strike">₹40</span>₹23</p>
+            <p class="title_item">Approx 500grams</p>
+            <button class="Btn_Cart" onclick="Cliq()">Add To Cart</button>
+             </div>
+          </div>
+      
+      
+         <div class="column">

```

Comment:
iterate the cards using javascript instead of writing every single card in separate div